commands/user.go

```diff
@@ -101,6 +101,15 @@ var ListUsersCmd = &cobra.Command{
 	Args:    cobra.NoArgs,
 }
 
+var VerifyUserCmd = &cobra.Command{
```

Comment:
What about using `user email verify` as a command? // @mgdelacroix

Reply:
I think I prefer `user verify-email` (for example) rather than creating an `email` subcommand

Reply:
I think we should maintain `verify` subcommand here. See [mattermost-cli](https://docs.mattermost.com/administration/command-line-tools.html?highlight=command%20line#mattermost-user-verify). For v6, maybe we can add this change to proposal?